### ISSUE TITLE
raiplay commit

### ISSUE THREAD

**Issue by dkwo** [blackjack4494/yt-dlc#134]
When trying to download from raiplay.it with yt-dl 2020.11.01.1, it does not work:
```
[RaiPlay] xxx: Downloading video JSON
ERROR: xxx: Failed to parse JSON (caused by JSONDecodeError('Expecting value: line 1 column 1 (char 0)')).
```
It seems there was a PR on yt-dl github organization that fixes this, but I'm not able to access it.
Does yt-dlc already include this commit, or if anyone has a copy of the commit would it be possible to merge it?

Thanks.

**Comment by nixxo** [comment]
I will look into it.

**Comment by dkwo** [comment]
Wonderful.
The url to test is https://www.raiplay.it/video/2020/11/Virus-e-segreti-di-Stato---Report---02-11-2020-eff77ea3-61bf-4d85-b18d-7e03fe5d90d2.html
Can you confirm that it works with yt-dlc?
Thank you.

**Comment by nixxo** [comment]
I can confirm it works for me.

> ydl https://www.raiplay.it/video/2020/11/Virus-e-segreti-di-Stato---Report---02-11-2020-eff77ea3-61bf-4d85-b18d-7e03fe5d90d2.html -F
>[RaiPlay] eff77ea3-61bf-4d85-b18d-7e03fe5d90d2: Downloading video JSON
>[RaiPlay] eff77ea3-61bf-4d85-b18d-7e03fe5d90d2: Downloading XML metadata for platform mon
>[RaiPlay] eff77ea3-61bf-4d85-b18d-7e03fe5d90d2: Downloading m3u8 information
>[RaiPlay] eff77ea3-61bf-4d85-b18d-7e03fe5d90d2: Downloading XML metadata for platform flash
>[RaiPlay] eff77ea3-61bf-4d85-b18d-7e03fe5d90d2: Downloading XML metadata for platform native
>[info] Available formats for eff77ea3-61bf-4d85-b18d-7e03fe5d90d2:
>format code  extension  resolution note
>hls-1892     mp4        928x522    1892k , avc1.77.30, mp4a.40.2
>hls-2524     mp4        1024x576   2524k , avc1.77.30, mp4a.40.2 (best)

**Comment by dkwo** [comment]
Thanks a lot, it works for me as well.
